### ISSUE TITLE
The type definition of `$loading` in Nuxt.js conflicts with element-ui

### ISSUE THREAD

**Issue by JohannesLichtenberger** [sirixdb/sirix-web-frontend#18]
See: https://github.com/nuxt/typescript/issues/49

I did the workaround locally mentioned in

https://github.com/nuxt/typescript/issues/49#issuecomment-531086770

However, I hope that gets fixed. Either from the Element-UI or Next.js developers...

**Comment by simhol** [comment]
I didn't check if there is an ongoing issue for this in the Element Repo. But I don't think it will be fixed by Nuxt

**Comment by JohannesLichtenberger** [comment]
Yeah, it already has the `wontfix` label ;-)

**Comment by JohannesLichtenberger** [comment]
Filed an issue, let's see: https://github.com/ElemeFE/element/issues/17630

**Comment by pareshjoshi** [comment]
But how do you workaround this? With latest sync, yarn build is now failing

**Comment by JohannesLichtenberger** [comment]
Bad workaround I guess (changing the type to any): https://github.com/nuxt/typescript/issues/49#issuecomment-531086770

**Comment by JohannesLichtenberger** [comment]
I hope it works for you, too? But it's for sure no real solution. For me as a Java and little bit Kotlin software engineer it's a bit strange to have such problems with name/type clashes.

**Comment by pareshjoshi** [comment]
Yes, that worked. But the repo it in broken state, unless it is documented in readme file, no one will be able to figure it out

**Comment by JohannesLichtenberger** [comment]
Yes, added to the README. However, I'm not sure if either only of the projects will change anything :(
